worlds/stardew_valley/content/mods/sve.py

```diff
@@ -38,19 +50,49 @@ def villager_hook(self, content: StardewContent):
             # Remove Lance if Ginger Island is not in content since he is first encountered in Volcano Forge
             content.villagers.pop(villagers_data.lance.name)
 
+    def harvest_source_hook(self, content: StardewContent):
+        content.untag_item(SVESeed.shrub, tag=ItemTag.CROPSANITY_SEED)
+        content.untag_item(SVESeed.fungus, tag=ItemTag.CROPSANITY_SEED)
+        content.untag_item(SVESeed.slime, tag=ItemTag.CROPSANITY_SEED)
+        content.untag_item(SVESeed.stalk, tag=ItemTag.CROPSANITY_SEED)
+        content.untag_item(SVESeed.void, tag=ItemTag.CROPSANITY_SEED)
+        content.untag_item(SVESeed.ancient_fern, tag=ItemTag.CROPSANITY_SEED)
+        if ginger_island_content_pack.name not in content.registered_packs:
+            # Remove Highlands seeds as these are behind Lance existing.
+            content.source_item(SVESeed.void)
+            content.source_item(SVEVegetable.void_root)
+            content.source_item(SVESeed.stalk)
+            content.source_item(SVEFruit.monster_fruit)
+            content.source_item(SVESeed.fungus)
+            content.source_item(SVEVegetable.monster_mushroom)
+            content.source_item(SVESeed.slime)
+            content.source_item(SVEFruit.slime_berry)
```

Comment:
I think those statements don't do anything. My understanding is that `source_item` add sources to an item, but in that case no source is given, so it does nothing.

If the intention is to remove the items, the best solution would be to pop them from `content.game_items`.